### ISSUE TITLE
Add `issue` option to `hit new` command

### ISSUE THREAD

**Issue by chshersh** [kowainik/hit-on#65]
Sometimes when you work on something, you don't have an issue for that. But you realize it only after you did all work. It would be really convenient to have a command that allows you to create issue from your terminal. Something like:

```shell
hit new-issue "Implement urgent fix for the image processing"
```

This command should create issue on GitHub and output the number of created issue to your terminal which you can use later with `hit new` command.

P.S. Probably better name than `new-issue` should be used.

**Comment by chshersh** [comment]
I wonder, how far in terms of what we can specify via CLI we should go. The current proposal suggests only issue title (which should be fine for most cases). And I'm not sure about labels and body and assignees... We have a separate issue for assigning to issue. And if you want to configure issue a lot, probably you can do this via the Web interface. This command is for quick issue creation.

Alternative syntax:

```haskell
hit new --issue "Title goes here"
```

Issue #72 proposed to create issue during switch. So you will write:

```haskell
hit new fix-docs -i
```

And this command will create an issue and switch you to a new branch called `<login>/fix-docs`. But what is the title of created issue should be? Should we split by `-` and uppercase first character of the first word so `fix-docs` will transform to `Fix docs`?

**Comment by bangn** [comment]
I would like to have a look.

With `git` there is no command to create new issue. [gihub/hub](https://github.com/github/hub) does have such command, however, would we like to have `hub` as a binary dependency?

**Comment by chshersh** [comment]
@bangn In `hit-on` we're using Haskell library `github` which implements Rest API for GitHub. This library should have functions for creating GitHub issues :slightly_smiling_face: We are using `hub` in `summoner` to create repositories, but since we already depend on `github` there's no need to use `hub` here.

* http://hackage.haskell.org/package/github
